packages/website/src/types.d.ts

```diff
@@ -1,11 +1,19 @@
-import 'process';
+// TODO: remove after v2.0.0-beta.10
+declare module '*.svg' {
+  import type { ComponentType, SVGProps } from 'react';
+
+  const ReactComponent: ComponentType<SVGProps<SVGSVGElement>>;
+
+  // eslint-disable-next-line import/no-default-export
+  export default ReactComponent;
+}
 
 declare global {
-  namespace NodeJS {
-    interface ProcessEnv {
+  declare const process: {
+    env: {
       TS_VERSION: string;
       ESLINT_VERSION: string;
       TS_ESLINT_VERSION: string;
-    }
-  }
+    };
+  };
```

Comment:
won't this override the global variable type instead of merging the type?

Reply:
we do not have global declaration anymore, i removed @types/node from `types` as this is used within context of browser